src/EnsembleKalmanInversions.jl

```diff
@@ -368,17 +390,86 @@ end
 
 """ Return a BitVector indicating whether the norm of the forward map
 for a given particle exceeds the median by `mrn.minimum_relative_norm`."""
-function (mrn::NormExceedsMedian)(G)
+function (mrn::NormExceedsMedian)(X, G, eki)
```

Comment:
so this change requires some of the tests in `test_ensemble_kalman_inversion.jl` to be updated. @glwagner can you do this? is it simply `@test sum(norm_exceeds_median(G)) == 0` -> `@test sum(norm_exceeds_median(θ, G, ski)) == 0`?